examples/ex6p.cpp

```diff
@@ -2,19 +2,19 @@
 //
 // Compile with: make ex6p
 //
-// Sample runs:  mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 1 -cs
-//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 2 -cs
-//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 2
+// Sample runs:  mpirun -np 4 ex6p -m ../data/star-hilbert.mesh -o 2
+//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -rm 1 -o 1
+//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -rm 1 -o 2
+//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -rm 1 -o 2 -h1
 //               mpirun -np 4 ex6p -m ../data/square-disc-nurbs.mesh -o 2
-//               mpirun -np 4 ex6p -m ../data/star.mesh -o 3
-//               mpirun -np 4 ex6p -m ../data/escher.mesh -o 2 -cs
-//               mpirun -np 4 ex6p -m ../data/escher.mesh -o 2
 //               mpirun -np 4 ex6p -m ../data/fichera.mesh -o 2
+//               mpirun -np 4 ex6p -m ../data/escher.mesh -rm 2 -o 2
+//               mpirun -np 4 ex6p -m ../data/escher.mesh -o 2 -cs
```

Comment:
This diff shows re-ordering of the sample runs similar to what was done in #2105. Any idea why it is showing like this?

When I diff this locally with
```console
git diff master...ex6p-runs2-dev
```
I get:
```diff
diff --git a/examples/ex6p.cpp b/examples/ex6p.cpp
index 17a694a94..14f156fd2 100644
--- a/examples/ex6p.cpp
+++ b/examples/ex6p.cpp
@@ -2,19 +2,19 @@
 //
 // Compile with: make ex6p
 //
-// Sample runs:  mpirun -np 4 ex6p -m ../data/star.mesh -o 3
-//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 1 -cs
-//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 2 -cs
-//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -o 2
+// Sample runs:  mpirun -np 4 ex6p -m ../data/star-hilbert.mesh -o 2
+//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -rm 1 -o 1
+//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -rm 1 -o 2
+//               mpirun -np 4 ex6p -m ../data/square-disc.mesh -rm 1 -o 2 -h1
 //               mpirun -np 4 ex6p -m ../data/square-disc-nurbs.mesh -o 2
 //               mpirun -np 4 ex6p -m ../data/fichera.mesh -o 2
-//               mpirun -np 4 ex6p -m ../data/escher.mesh -o 2
+//               mpirun -np 4 ex6p -m ../data/escher.mesh -rm 2 -o 2
 //               mpirun -np 4 ex6p -m ../data/escher.mesh -o 2 -cs
 //               mpirun -np 4 ex6p -m ../data/disc-nurbs.mesh -o 2
 //               mpirun -np 4 ex6p -m ../data/ball-nurbs.mesh
 //               mpirun -np 4 ex6p -m ../data/pipe-nurbs.mesh
 //               mpirun -np 4 ex6p -m ../data/star-surf.mesh -o 2
-//               mpirun -np 4 ex6p -m ../data/square-disc-surf.mesh -o 2 -cs
+//               mpirun -np 4 ex6p -m ../data/square-disc-surf.mesh -rm 2 -o 2
 //               mpirun -np 4 ex6p -m ../data/inline-segment.mesh -o 1 -md 200
 //               mpirun -np 4 ex6p -m ../data/amr-quad.mesh
 //               mpirun -np 4 ex6p --restart
```

Reply:
I tried merging master into this branch, seems fixed now.